content/docs/command-reference/import-url.md

```diff
@@ -102,7 +102,7 @@ $ dvc run -d https://example.com/path/to/data.csv \
           wget https://example.com/path/to/data.csv -O data.csv
 ```
 
-Both methods generate a [DVC-files](/doc/user-guide/dvc-file-format) with an
+Both methods generate a [`.dvc` files](/doc/user-guide/dvc-file-format) with an
 external dependency, but the one created by `dvc import-url` preserves the
```

Comment:
This one is tricky...

Reply:
`dvc import-url` generates an import stage (`.dvc` file [linked] ) and `dvc run` a regular stage (in `dvc.yaml` [linked] ). Both have an external dependency, but the one created by `dvc import-url` preserves...